fastlane/Fastfile

```diff
@@ -91,6 +70,22 @@ platform :ios do
             changelog: changelog_from_git_commits)
     end
 
+    lane :update_version_number do |options|
+        version_number_string = options[:version_number_string]
+        increment_version_number(
+            version_number: version_number_string,
+            xcodeproj: iOSProject
+        )
+        increment_build_number(
+            build_number: "#{get_datetime_for_build_version}.#{number_of_commits}",
+            xcodeproj: iOSProject
+        )
+        # Git
```

Comment:
I think this comment is either not adding anything and can be removed or needs expanding